model/gen-ai/registry.yaml

```diff
@@ -311,3 +316,21 @@ groups:
         type: string
         brief: A fingerprint to track any eventual change in the Generative AI environment.
         examples: ["fp_44709d6fcb"]
+  - id: registry.gen_ai.aws.bedrock
+    type: attribute_group
+    display_name: Amazon Bedrock Attributes
+    brief: >
+        This document defines attributes for AWS Bedrock.
+    attributes:
+      - id: gen_ai.aws.bedrock.guardrail.id
+        type: string
+        stability: experimental
+        brief: >
+          The unique identifier of the AWS Bedrock Guardrail. A [guardrail](https://docs.aws.amazon.com/bedrock/latest/userguide/guardrails.html) helps safeguard and prevent unwanted behavior from model responses or user messages.
+        examples: ["sgi5gkybzqak"]
+      - id: gen_ai.aws.bedrock.knowledge_base.id
```

Comment:
is there a difference between knowledge_base and data_source? do we need both ids?

Reply:
From AWS doc "A data source contains files or content with information that can be retrieved when your knowledge base is queried.", so two different ids sounds legit to track the source of the data when you have more than one source in your knowledge base.